catkin_ws/src/kpr_arm_control/src/arm_control/arm_control.cpp

```diff
@@ -37,13 +37,16 @@ const uint8_t CUT_OPEN_PIN = 0x3;
 
 bool pinstates[] = {false, false, true};
 #define PRESSURE_STATE 0
-///pinstates[PRESSURE_STATE];
 #define CUT_CLOSED_STATE 1
-///pinstates[CUT_CLOSED_STATE];
 #define CUT_OPEN_STATE 2
-///pinstates[CUT_OPEN_STATE];
 
+const bool expectedValues[] = {true, false, true, true};
+#define EXPECT_VAC 0
+#define EXPECT_NOVAC 1
+#define EXPECT_CUT_OPEN 2
+#define EXPECT_CUT_CLOSED 3
```

Comment:
Why not just define these as true or false? This would remove the need for the `expectedValues` array.